nodejs/week2/homework/warmup/index.js

```diff
@@ -0,0 +1,18 @@
+const express = require("express");
+const app = express();
+
+app.get("/", (req, res) => res.send("nodejs week2 homework"));
+
+app.get('/numbers/add', (req, res) => {
+    const first = parseInt(req.query.first);
+    const second = parseInt(req.query.second);
+    res.send(`sum: ${first + second}`);
+});
+
+app.get('/numbers/multiply/:first/:second', (req, res) => {
```

Comment:
The code would not work correctly if the parameters are not numbers.
GET /numbers/multiply/abbv/ghhd